### DISCUSSION TITLE
Decaffeinate

### DISCUSSION THREAD

**Discussion by fabianfiorotto** [pulsar-edit/.github#43] [General]
Atom team was in progress to remove CoffeeScript from the sources but they never finished the work. I can help with that if you want. there are tons of small classes like this one that are easy to migrate 

https://github.com/pulsar-edit/pulsar/blob/0732c5430fbf059dde8a019e10986cfd813e6c74/src/token.coffee

**Comment by confused-Techie** [comment]
Speaking with @mauricioszabo and myself decaffeination on the codebase is something we support. If this is something you would like to work on, its something the two of us are on board with.
